syft/execution/communication.py

```diff
@@ -17,6 +18,7 @@ class CommunicationAction(Action):
     def __init__(
         self,
         obj_id: Union[str, int],
+        name: str,  # @TODO: maybe refactor into enum later?
```

Comment:
Yeah, an `enum` might be a good idea. On `ComputationAction`, there's such a wide variety of methods (from PyTorch and Tensorflow) that can be captured that it didn't seem worth it. Here, since we're only dealing with PySyft methods, it might make more sense.